lib/views/auth_page.dart

```diff
@@ -1,12 +1,126 @@
 import 'package:cycle_lock/controllers/login_controller.dart';
+import 'package:cycle_lock/views/main_page.dart';
 import 'package:flutter/material.dart';
 import 'package:get/get_state_manager/get_state_manager.dart';
+import 'package:get/get.dart';
 
 class AuthPage extends GetView<AuthController> {
   const AuthPage({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Container(
+      decoration: const BoxDecoration(
+          image: DecorationImage(
+              image: AssetImage('assets/login bg.png'), fit: BoxFit.cover)),
+      child: Scaffold(
+        backgroundColor: Colors.transparent,
+        body: Stack(
+          children: [
+            Container(
+              padding: const EdgeInsets.only(left: 40, top: 265),
+              child: const Text(
+                'Welcome User',
+                style: TextStyle(
+                    color: Colors.black,
+                    fontWeight: FontWeight.bold,
+                    fontFamily: 'Moon',
+                    fontSize: 40),
+              ),
+            ),
+            SingleChildScrollView(
+              child: Container(
+                padding: EdgeInsets.only(
+                    top: MediaQuery.of(context).size.height * 0.5,
+                    right: 35,
+                    left: 35),
+                child: Column(
+                  children: [
+                    TextField(
+                      decoration: InputDecoration(
+                        fillColor: Colors.grey.shade50,
+                        filled: true,
+                        hintText: 'Enter your e-mail',
+                        border: OutlineInputBorder(
+                            borderRadius: BorderRadius.circular(10)),
+                      ),
+                    ),
+                    const SizedBox(
+                      height: 37,
+                    ),
+                    TextField(
```

Comment:
Since both the TextField widgets share many similar attributes, create a customTextField widget inside widgets folder with the necessary constructor parameters and import it to this widget and use it.

Reduces code duplication and increases code maintainability